source/features/infinite-scroll.tsx

```diff
@@ -3,16 +3,16 @@ import debounce from 'debounce-fn';
 import features from '../libs/features';
 import observeEl from '../libs/simplified-element-observer';
 
-let btn: HTMLButtonElement;
+let btn: HTMLButtonElement | undefined;
```

Comment:
```suggestion
let button: HTMLButtonElement | undefined;
```

Reply:
Didn't you add a rule to detect all of these shortened variables? e.g. `evt` and `err`

Reply:
Yes, but I disabled it in XO until the rule stabilizes and I have time to deal with the repercussions (it will be unpopular with many users). We could enable it manually in our config for now though?

Reply:
Done. Love it. However `--fix` drops TypeScript types

Reply:
> However --fix drops TypeScript types

That's not good...